pkg/apis/autoscaling/register.go

```diff
@@ -41,15 +41,25 @@ const (
 	// HPA is Kubernetes Horizontal Pod Autoscaler
 	HPA = "hpa.autoscaling.knative.dev"
 
-	// MinScaleAnnotationKey is the annotation to specify the minimum number of Pods
+	// DeprecatedMinScaleAnnotationKey is the annotation to specify the minimum number of Pods
 	// the PodAutoscaler should provision. For example,
 	//   autoscaling.knative.dev/min-scale: "1"
-	MinScaleAnnotationKey = GroupName + "/min-scale"
+	DeprecatedMinScaleAnnotationKey = GroupName + "/min-scale"
 
-	// MaxScaleAnnotationKey is the annotation to specify the maximum number of Pods
+	// ScaleMinAnnotationKey is the annotation to specify the minimum number of Pods
+	// the PodAutoscaler should provision. For example,
+	//   autoscaling.knative.dev/scale-min: "1"
+	ScaleMinAnnotationKey = GroupName + "/scale-min"
+
+	// DeprecatedMaxScaleAnnotationKey is the annotation to specify the maximum number of Pods
 	// the PodAutoscaler should provision. For example,
 	//   autoscaling.knative.dev/max-scale: "10"
-	MaxScaleAnnotationKey = GroupName + "/max-scale"
+	DeprecatedMaxScaleAnnotationKey = GroupName + "/max-scale"
+
+	// ScaleMaxAnnotationKey is the annotation to specify the maximum number of Pods
+	// the PodAutoscaler should provision. For example,
+	//   autoscaling.knative.dev/scale-max: "10"
+	ScaleMaxAnnotationKey = GroupName + "/scale-max"
```

Comment:
Do we really need to rename all these variables?  It seems like a simpler change would be update 'MaxScaleAnnotationKey` to the correct value, then add the deprecated key to `MaxScaleAnnotation`.

Picking this one as an example, but similar comments for the updates of `MaxScale` -> `ScaleMax`, etc.

That said, I could be persuaded otherwise...